modules/imgproc/src/opencl/boxFilter3x3.cl

```diff
@@ -0,0 +1,127 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+
+__kernel void boxFilter3x3_8UC1_cols16_rows2(__global const uint* src, int src_step,
+                                             __global uint* dst, int dst_step, int dst_offset, int rows, int cols
```

Comment:
There are [missing checks](https://github.com/opencv/opencv/pull/7500#discussion_r83574477) for src_step/dst_step